Agrume/Agrume.swift

```diff
@@ -527,29 +556,32 @@ extension Agrume: AgrumeCellDelegate {
     _spinner = nil
   }
 
-  func dismissAfterFlick() {
+	@objc func dismissAfterFlick() {
     UIView.animate(withDuration: Agrume.transitionAnimationDuration,
                    delay: 0,
                    options: .beginFromCurrentState,
                    animations: { [unowned self] in
                     self.collectionView.alpha = 0
+										self.closeBtn.alpha = 0
                     self.blurContainerView.alpha = 0
       }, completion: dismissCompletion)
   }
   
   func dismissAfterTap() {
+		if(shouldDismissWithTap) {
```

Comment:
Adding the conditional here means that the function name is a lie now 😬 – this bool should rather go to the cells, they decide if they call the delegate or not.